<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Droid</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 4px solid #61136E;
    margin-top: -2px;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    border: 2px solid #61136E;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    padding-top: 3px;
    padding-bottom: 3px;
    padding-left: 10px;
    padding-right: 10px;
}

QTabBar::tab:selected {
    background: #61136E;
    color: #FFFFFF;
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
    color: #000000
}x</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="status">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>status</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>801</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="semiButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Droid</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:checked {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
         </property>
         <property name="text">
          <string>semi-auto</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">modeButtons</string>
         </attribute>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="burstButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Droid</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:checked {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
         </property>
         <property name="text">
          <string>burst: 3</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">modeButtons</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="autoButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Droid</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:checked {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
         </property>
         <property name="text">
          <string>automatic</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">modeButtons</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSlider" name="burstSlider">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>331</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #000000;
    height: 4px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: #000000;
    margin: 5px;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #9313a0, stop:1 #2f133c);
    border: 1px solid #000000;
    width: 18px;
    margin: -16px 2px;
    border-radius: 3px;
}</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="sliderPosition">
       <number>3</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="fpsLCD">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>320</width>
        <height>130</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid #61136e;
border-radius: 8px;
color: #000000</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="FPSupButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Webdings</family>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    padding-left:4px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:pressed {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FPSdownButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Webdings</family>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    padding-left:4px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:pressed {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="ammoLCD">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>320</width>
        <height>130</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid #61136e;
border-radius: 8px;
color: #000000</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="FPS">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Droid</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000</string>
      </property>
      <property name="text">
       <string>FPS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="AMMO">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Droid</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000</string>
      </property>
      <property name="text">
       <string>AMMO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLCDNumber" name="psiLCD">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>300</y>
        <width>331</width>
        <height>130</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 3px solid #61136e;
border-radius: 8px;
color: #000000</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="ROCKETPRESSURE">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Droid</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000</string>
      </property>
      <property name="text">
       <string>Rocket Pressure</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="PSIdownButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Webdings</family>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    padding-left:4px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:pressed {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PSIupButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Webdings</family>
        <pointsize>64</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    padding-left:4px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:pressed {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="laserButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>300</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Droid</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:checked {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>laser</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="lightButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>380</y>
        <width>181</width>
        <height>50</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Droid</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: 3px solid #61136E;
    border-radius: 8px;
    background-color: #FFFFFF;
	color: #000000;
}

QPushButton:checked {
    background-color: #61136E;
	color: #FFFFFF;
}</string>
      </property>
      <property name="text">
       <string>flashlight</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="led">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>led settings</string>
     </attribute>
     <widget class="QLabel" name="blasterImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>75</y>
        <width>800</width>
        <height>288</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>blaster.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="data">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>data logs</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="modeButtons"/>
 </buttongroups>
</ui>
